docs/methods.rst

```diff
@@ -39,17 +39,21 @@ update
 
 Arguments:
 
+* date (String|Date|Array, optional)
 * date (String|Date, optional)
```

Comment:
Well I wasn't sure how to show the fact that date is `*args` for `string` and `Date`. But not when `Array`.